docs/models.rst

```diff
@@ -127,19 +135,29 @@ helper table that is used for the relationship.  For this helper table it
 is strongly recommended to *not* use a model but an actual table::
 
     tags = db.Table('tags',
-        db.Column('tag_id', db.Integer, db.ForeignKey('tag.id')),
-        db.Column('page_id', db.Integer, db.ForeignKey('page.id'))
+        db.Column('tag_id', db.Integer, db.ForeignKey('tag.id'), nullable=False),
+        db.Column('page_id', db.Integer, db.ForeignKey('page.id'), nullable=False)
```

Comment:
Usually I mark both columns as `primary_key=True`. Same comment about not needing type applies as well.